sympy/parsing/tests/test_latex_lark.py

```diff
@@ -0,0 +1,272 @@
+from sympy.testing.pytest import raises, XFAIL
+from sympy.external import import_module
+
+from sympy.concrete.products import Product
+from sympy.concrete.summations import Sum
+from sympy.core.add import Add
+from sympy.core.function import (Derivative, Function)
+from sympy.core.mul import Mul
+from sympy.core.numbers import (E, oo)
+from sympy.core.power import Pow
+from sympy.core.relational import (GreaterThan, LessThan, StrictGreaterThan, StrictLessThan, Unequality)
+from sympy.core.symbol import Symbol
+from sympy.functions.combinatorial.factorials import (binomial, factorial)
+from sympy.functions.elementary.complexes import (Abs, conjugate)
+from sympy.functions.elementary.exponential import (exp, log)
+from sympy.functions.elementary.integers import (ceiling, floor)
+from sympy.functions.elementary.miscellaneous import (root, sqrt)
+from sympy.functions.elementary.trigonometric import (asin, cos, csc, sec, sin, tan)
+from sympy.integrals.integrals import Integral
+from sympy.series.limits import Limit
+
+from sympy.core.relational import Eq, Ne, Lt, Le, Gt, Ge
+from sympy.physics.quantum.state import Bra, Ket
+from sympy.abc import x, y, z, a, b, c, t, k, n
+lark = import_module("lark")
+
+if not lark:
+    disabled = True
```

Comment:
is this variable needed? Correct way to use it would be `disabled = lark is None`

Reply:
I'm not sure if it is needed or not. This is how ANTLR tests are disabled if the `antlr4-python3-runtime` is not found, in the current [master](https://github.com/sympy/sympy/blob/46558492a74bf2a324f9560af008b6c3ee587a6c/sympy/parsing/tests/test_latex.py#L27-L29).

Since we want to do the same thing if Lark is not found, I added it.

Reply:
This is checked for in [this function](https://github.com/sympy/sympy/blob/0ec134d277f14961e8b6629182895fc986658c2e/sympy/conftest.py#L67-L74).